versions/datastax/3.24.0/patch.patch

```diff
@@ -0,0 +1,156 @@
++                    CCM_CLUSTER = CCMScyllaCluster(path, cluster_name, **ccm_options)
++                    CCM_CLUSTER.set_configuration_options({'experimental_features': ['lwt', 'udf'], 'start_native_transport': True})
++                else:
++                    CCM_CLUSTER = CCMCluster(path, cluster_name, **ccm_options)
++                    CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
+                 if Version(cassandra_version) >= Version('2.2'):
+                     CCM_CLUSTER.set_configuration_options({'enable_user_defined_functions': True})
+                     if Version(cassandra_version) >= Version('3.0'):
+@@ -570,18 +584,18 @@
+                             })
+                 common.switch_cluster(path, cluster_name)
+                 CCM_CLUSTER.set_configuration_options(configuration_options)
+-                CCM_CLUSTER.populate(nodes, ipformat=ipformat, use_single_interface=use_single_interface)
++                CCM_CLUSTER.populate(nodes, ipformat=ipformat)
+ 
+     try:
+         jvm_args = []
+ 
+         # This will enable the Mirroring query handler which will echo our custom payload k,v pairs back
+ 
+-        if 'graph' in workloads:
+-            jvm_args += ['-Xms1500M', '-Xmx1500M']
+-        else:
+-            if PROTOCOL_VERSION >= 4:
+-                jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
++        # if 'graph' in workloads:
++        #     jvm_args += ['-Xms1500M', '-Xmx1500M']
++        # else:
++        #     if PROTOCOL_VERSION >= 4:
++        #         jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
+         if len(workloads) > 0:
+             for node in CCM_CLUSTER.nodes.values():
+                 node.set_workloads(workloads)
+Index: tests/integration/standard/test_authentication_misconfiguration.py
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/tests/integration/standard/test_authentication_misconfiguration.py b/tests/integration/standard/test_authentication_misconfiguration.py
+--- a/tests/integration/standard/test_authentication_misconfiguration.py	(revision c88255f202a21bbbae35f16e603b0f10f2f2cf36)
++++ b/tests/integration/standard/test_authentication_misconfiguration.py	(date 1641374958430)
+@@ -19,6 +19,12 @@
+ 
+ class MisconfiguredAuthenticationTests(unittest.TestCase):
+     """ One node (not the contact point) has password auth. The rest of the nodes have no auth """
++    # TODO: 	Fix ccm to apply following options to scylla.yaml
++    # 	node3.set_configuration_options(values={
++    # 	'authenticator': 'PasswordAuthenticator',
++    # 	'authorizer': 'CassandraAuthorizer',
++    # 	})
++    # To make it working for scylla
```

Comment:
Can you please explain this?

Reply:
This is the code from our python-driver repo.
We run this matrix for the following version:
* scylla-4.4
*scylla-4.5
* scylla-4.6
* master
Do you want me to remove it?

Reply:
Remove it

Reply:
@dkropachev 
Do you remember why you wrote it?
The test was passed for the Scylla master with protocol 4.